app/src/docs/components/PageTurner/index.jsx

```diff
@@ -3,82 +3,108 @@
 import React from 'react'
 import { Link, withRouter, type Location } from 'react-router-dom'
 import cx from 'classnames'
-import { formatPath } from '$shared/utils/url'
 import SvgIcon from '$shared/components/SvgIcon'
+import { docsNav } from '$docs/components/DocsLayout/Navigation/navLinks'
 
-import type { NavigationLink } from '$docs/flowtype/navigation-types'
+import type { DocsNav } from '$docs/flowtype/navigation-types'
 
 import styles from './pageTurner.pcss'
 
 type Props = {
-    navigationItems: NavigationLink,
     location: Location,
 }
 
 class PageTurner extends React.Component<Props> {
```

Comment:
Not sure but I wonder if the built-in utilities in `react-router` could be of assistance here:  e.g. https://reacttraining.com/react-router/web/api/matchPath

Reply:
I didn't think of those at the time, after a quick glance I'd say, *probably* but I don't think there's much value in this particular refactor.